aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -186,15 +186,7 @@ For more information, see [Problem details service](xref:web-api/handle-errors##
 
 ### Route groups
 
-The <xref:Microsoft.AspNetCore.Builder.EndpointRouteBuilderExtensions.MapGroup%2A> extension method helps organize groups of endpoints with a common prefix. It reduces repetitive code and allows for customizing entire groups of endpoints with a single call to methods like <xref:Microsoft.AspNetCore.Builder.AuthorizationEndpointConventionBuilderExtensions.RequireAuthorization%2A> and <xref:Microsoft.AspNetCore.Builder.RoutingEndpointConventionBuilderExtensions.WithMetadata%2A>.
-
-For example, the following code creates two similar groups of endpoints:
-
-:::code language="csharp" source="~/fundamentals/minimal-apis/7.0-samples/todo-group/Program.cs" id="snippet_MapGroup":::
-
-:::code language="csharp" source="~/fundamentals/minimal-apis/7.0-samples/todo-group/TodoEndpoints.cs" id="snippet_TodoEndpoints":::
-
-For more information, see [Route groups](xref:fundamentals/minimal-apis#route-groups)
+[!INCLUDE[](~/includes/route-groups.md)]
```

Comment:
It doesn't make sense to include the entirety of the route groups doc in the release notes...

Reply:
Do you really think so? This seems smaller than the "OpenAPI improvements for minimal APIs" section and I consider it a similarly big feature. And it's a feature that needs to be demonstrated in conjunction of something else like endpoint filters.

That said, it'd be very easy to revert the changes in this file if people generally agree this is too much detail.